plugins/google_analytics/girder_google_analytics/web_client/main.js

```diff
@@ -5,21 +5,26 @@ import './lib/backbone.analytics';
 import './routes';
 
 events.on('g:appload.after', function () {
+    var dnt = navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack;
     /* eslint-disable */
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-    /* eslint-enable */
+    if (dnt != "1" && dnt != "yes") {
```

Comment:
Can we use strict equality checking here for clarity?

Reply:
Sure. For Firefox and Chrome on my system, I can verify that the the value is `"1"`.